lib/Importer/ProtobufLoader.cpp

```diff
@@ -35,6 +35,16 @@ bool isArrayConstant(llvm::ArrayRef<size_t> a) {
   return true;
 }
 
+Expected<int> getPositiveAxis(int axis, int rank) {
+  RETURN_ERR_IF_NOT(-rank <= axis, "Axis parameter invalid!");
+  RETURN_ERR_IF_NOT(axis < rank, "Axis parameter invalid!");
```

Comment:
Can you add axis and rank to the messages with strFormat

Reply:
Done.